scripts/ci/upload-benchmark

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+"""Script to upload benchmark results to an s3 location."""
+import os
+import argparse
+from subprocess import check_call
+
+DEFAULT_BUCKET = os.environ.get('PERF_RESULTS_BUCKET')
+
+
+def main(args):
+    source = args.directory
+    destination = args.s3_location
+    check_call('aws s3 cp %s %s' % (source, destination), shell=True)
```

Comment:
I was trying this out and I was expecting to be able to upload the results directory.  However there's no `--recursive` here.  What did you have in mind for how this script would be used?